CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Introduction
+
+The purpose of this document is to show contributors to the Modernisation Platform code the best practices and testing that should be undertaken. The code should be in Terraform and tests, if coded, in Go.
+
+## Code of Conduct
+
+The Ministry of Justice [Code of Conduct](https://github.com/ministryofjustice/.github/blob/main/CODE_OF_CONDUCT.md) is used to govern the project and those participating.
+Please report unacceptable behavior to the maintainers or administrators.
+
+## Standards
+
+We follow the [MoJ Technical Guidance](https://technical-guidance.service.justice.gov.uk/#moj-technical-guidance),
+which complements the [Service Standard](https://www.gov.uk/service-manual/service-standard) ,
+and also the MoJ [Cyber and Technical Security Guidance](https://security-guidance.service.justice.gov.uk/#cyber-and-technical-security-guidance).
+
+# Modernisation Platform Documentation
+To access the modernisation documentation follow this [MP Documentation](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform).
+For the general terraform you can access [here](https://user-guide.modernisation-platform.service.justice.gov.uk/runbooks/terraform.html)
+If you are interested in running Terraform locally try [Running Terraform Plan Locally](https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/running-terraform-plan-locally.html#running-terraform-plan-locally)
+
+These can be located by running a search in the documentation [MP Documentation](https://user-guide.modernisation-platform.service.justice.gov.uk/#modernisation-platform). 
+
+# Coding examples
+
+There are several examples in this repository of coding and also additional ones in the modernisation-platform-environments repository. One, which we recommend to users to examine, is the example code in moderisation-platform-environments/terraform/environments/example files. This has samples of code showing how to build ec2, security groups, load balancers, rds and others. These are named well and the code can be copied and pasted into your own code so they are available locally. When these are copied the names should be changed so they do not contain the example names.
```

Comment:
This is more relevant to users than to people contributing, I think this should say something along the lines of if creating a module and want to show it working or if you wish to provide and example of code which can be used on the platform, this folder can be used to do that, and mention about keeping the code in distinct sections so that they can be run independantly.